build-scripts/ubuntu-1604/prepare-package.sh

```diff
@@ -35,8 +35,6 @@ sed -i -r "s~sovrin-dev~sovrin~" "$repo/setup.py"
 
 echo -e "\n\nPrepare debian package versions"
 sed -i -r "s~indy-node==([0-9\.]+[0-9])(\.)?([a-z]+)~indy-node==\1\~\3~" "$repo/setup.py"
-sed -i -r "s~sovtoken==([0-9\.]+[0-9])(\.)?([a-z]+)~sovtoken==\1\~\3~" "$repo/setup.py"
```

Comment:
can't hold sovtoken, sovtokenfees, libsovtoken until the 20.04 versions are available.